runner/src/indexer-logger/indexer-logger.test.ts

```diff
@@ -127,5 +128,21 @@ describe('IndexerLogger', () => {
 
       expect(query).not.toHaveBeenCalled();
     });
+
+    it('log status for indexer', async () => {
+      const indexerLogger = new IndexerLogger(functionName, 5, mockDatabaseConnectionParameters, genericMockPgClient);
+      await indexerLogger.setIndexerStatus(IndexerStatus.RUNNING);
+      expect(query.mock.calls[0][0]).toEqual(
```

Comment:
```suggestion
      expect(query.mock).toBeCalledWith(
```
I believe this works too. Access the actual `calls` themselves is only really necessary if you need to check sequence or specific calls.